workers/data_refinery_workers/processors/management/commands/queue_processor.py

```diff
@@ -0,0 +1,47 @@
+from django.core.management.base import BaseCommand
```

Comment:
It will be nice if you can write some docstring at the very beginning to describe the purpose and usage of this management command (and any prerequisite if it has).